### ISSUE TITLE
Version 1.11 seems to be broken

### ISSUE THREAD

**Issue by alternatiph** [DanTheMan827/ios-app-signer#94]
Hi
It seems that version 1.11 does not sign frameworks inside the package and signed apps crash upon openning.

Sample crash report:
```
Exception Type:  EXC_CRASH (SIGABRT)
Exception Codes: 0x0000000000000000, 0x0000000000000000
Exception Note:  EXC_CORPSE_NOTIFY
Termination Description: DYLD, Library not loaded: @rpath/libswiftAVFoundation.dylib
Referenced from: /var/containers/Bundle/Application/B1EE55E6-.../TestApp.app/TestApp
Reason: no suitable image found. 
Did find:
/private/var/containers/Bundle/Application/B1EE55E6-.../TestApp.app/Frameworks/libswiftAVFoundation.dylib: code signature invalid for '/private/var/containers/Bundle/Application/B1EE55E6-.../TestApp.app/Frameworks/libswiftAVFoundation.dylib'
/private/var/containers/Bundle/Application/B1EE55E6-.../TestApp.app/Frameworks/libswiftAVFoundation.dylib: code signature invalid for '/private/var/containers/Bundle/Application/B1EE55E6-.../TestApp.app/Frameworks/libswiftAVFoundation.dylib'
/private/var/containers/Bundle/Application/B1EE55E6-.../TestApp.app/Frameworks/libswiftAVFoundation.dylib: stat() failed with errno=1
/private/var/containers/Bundle/Applicati
```

**Comment by Tommrodrigues** [comment]
Similar issue here. After signing a `.ipa` with v1.11 and trying to load it onto an iOS device using Xcode I get the following error:

```
The executable was signed with invalid entitlements.
```

I'm using v1.10 for now until the issue is fixed as I'm having no problems with the older version.

**Comment by spockers** [comment]
I sign iOS jailbreak apps for ad-hoc distribution and had to revert to v1.10 too. v1.11 works for all but one app, Chimera, which installs fine but crashes on launch. The same ipa when signed with v.1.10 runs without issue.

**Comment by DanTheMan827** [comment]
I'll have to look into this

it seems like dylibs aren't being signed?

that's a little odd because the app I tested with while coding was Kodi

**Comment by spockers** [comment]
@DanTheMan827, try [Chimera](https://chimera.sh/downloads/ios/1.0.6.ipa), it will not produce any errors, but it won't run either.

**Comment by DanTheMan827** [comment]
Fixed in d36e5e6e6dd0f38fb31e14ca231c1ce5f1883b0c
